server/asset/ltc/ltc.go

```diff
@@ -19,20 +19,23 @@ var (
 		ScriptHashAddrID: 0x32,
 		Bech32HRPSegwit:  "ltc",
 		CoinbaseMaturity: 100,
+		Net:              0xd9b4bef9,
```

Comment:
I think you used bitcoin's mainnet `wire.BitcoinNet` here. 

litecoin:

https://github.com/ltcsuite/ltcd/blob/4faa85afe0ded0dc7316ad0fa2912c3ae3507794/wire/protocol.go#L150

bitcoin:

https://github.com/btcsuite/btcd/blob/9f0179fd2c46caba343b6515602cf37172f14d5f/wire/protocol.go#L150

Reply:
Nice catch.